pages/profile/permission/index.js

```diff
@@ -20,10 +20,10 @@ module.exports = settings => {
   }));
 
   app.use('/', (req, res, next) => {
-    const hasRoles = !!(req.profile.roles && req.profile.roles.length);
-    const hasPil = !!(req.profile.pil && req.profile.pil.status === 'active' && req.profile.pil.establishmentId === req.establishmentId);
-    const hasProjects = !!(req.profile.projects && req.profile.projects.length && some(req.profile.projects, project => project.status === 'active'));
-    res.locals.static.isNamed = hasRoles || hasPil || hasProjects;
+    const hasRoles = some(req.profile.roles, role => role.establishmentId === req.establishmentId);
+    const hasPil = req.profile.pil && req.profile.pil.status === 'active' && req.profile.pil.establishmentId === req.establishmentId;
+    const hasProjects = some(req.profile.projects, project => project.status === 'active' && project.establishmentId === req.establishmentId);
+    res.locals.static.isNamed = !!hasRoles || !!hasPil || !!hasProjects;
```

Comment:
Aren't they all booleans already, so no need to cast here?

Reply:
They were casted before, so I left them casted, but you're right.

Reply:
https://github.com/UKHomeOffice/asl-pages/pull/716/commits/9bc423de09d3137f837d4abb8e73e9f537c33a21#diff-2c902969d60f43f430908e85a50cf87fL26 WERE THEY THO

Reply:
Jesus Lenny.

Reply:
LITERALLY YES!

<img width="76" alt="Screenshot 2020-02-27 at 16 34 48" src="https://user-images.githubusercontent.com/117398/75464471-271aae00-597f-11ea-88e5-66c1e600d246.png">